docs/elasticsearch/logs.md

```diff
@@ -0,0 +1,173 @@
+   ```
+
+   - `${sys:es.logs.base_path}` is the directory for logs (e.g. `/var/log/elasticsearch/`).
+   - `${sys:es.logs.cluster_name}` is the name of the cluster.
+   - `[%node_name]` is the name of the node.
+
+
+## Slow logs
+
+Elasticsearch has two *slow logs*, logs that help you identify performance issues: the search slow log and the indexing slow log.
+
+These logs rely on thresholds to define what qualifies as a "slow" search or indexing operation. For example, you might decide that a query is slow if it takes more than 15 seconds to complete. By default, both logs are disabled (all thresholds set to `-1`):
+
+```json
+GET <some-index>/_settings?include_defaults=true
+
+{
+  "indexing": {
+    "slowlog": {
+      "reformat": "true",
+      "threshold": {
+        "index": {
+          "warn": "-1",
+          "trace": "-1",
+          "debug": "-1",
+          "info": "-1"
+        }
+      },
+      "source": "1000",
+      "level": "TRACE"
+    }
+  },
+  "search": {
+    "slowlog": {
+      "level": "TRACE",
+      "threshold": {
+        "fetch": {
+          "warn": "-1",
+          "trace": "-1",
+          "debug": "-1",
+          "info": "-1"
+        },
+        "query": {
+          "warn": "-1",
+          "trace": "-1",
+          "debug": "-1",
+          "info": "-1"
+        }
+      }
+    }
+  }
+}
+```
+
+To enable these logs, increase one or more thresholds:
+
+```json
+PUT <some-index>/_settings
+{
+  "indexing": {
+    "slowlog": {
+      "threshold": {
+        "index": {
+          "warn": "15s",
+          "trace": "750ms",
+          "debug": "3s",
+          "info": "10s"
+        }
+      },
+      "source": "500",
+      "level": "INFO"
+    }
+  }
+}
+```
+
+- `reformat` specifies whether to log the document `_source` field as a single line (`true`) or let it span multiple lines (`false`).
+- `source` is the number of characters of the document `_source` field to log.
+- `level` is the minimum log level to include.
+
+A line from `elasticsearch_index_indexing_slowlog.log` might look like this:
+
+```
+node1 | [2019-10-24T19:48:51,012][WARN][i.i.s.index] [node1] [some-index/i86iF5kyTyy-PS8zrdDeAA] took[3.4ms], took_millis[3], type[_doc], id[1], routing[], source[{"title":"Your Name", "Director":"Makoto Shinkai"}]
+```
+
+Slow logs can consume considerable disk space if you set thresholds or levels too low. Consider enabling them temporarily for troubleshooting or performance tuning. To disable slow logs, return all thresholds to `-1`.
+
+
+## Deprecation logs
+
+Deprecation logs record when clients make deprecated API calls to your cluster. These logs can help you identify and fix issues prior to upgrading to a new major version. By default, Elasticsearch logs deprecated API calls at the WARN level, which works well for almost all use cases. If desired, configure `logger.deprecation.level` using `_cluster/settings`, `elasticsearch.yml`, or `log4j2.properties`.
```

Comment:
Maybe add a link to the CAT API section as another way to troubleshoot issues?

Reply:
Probably going to punt on this one. I just don't see a great spot to include it, and I think the CAT APIs give you more of a "which index is red?" vs. "why is my search failing?" view of the cluster.